source/get-started/connect-to-mongodb.txt

```diff
@@ -0,0 +1,84 @@
+.. _cpp-quick-start-connect-to-mongodb:
+
+==================
+Connect to MongoDB
+==================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: test connection, runnable, code example
+
+.. procedure::
+   :style: connected
+
+   .. step:: Create a project directory
+
+      In your shell, run the following command to create a directory called
+      ``cpp-quickstart`` for this project:
+
+      .. code-block:: bash
+
+         mkdir cpp-quickstart
+
+      Run the following command to navigate into the ``cpp-quickstart`` directory:
+
+      .. code-block:: bash
+
+         cd cpp-quickstart
+
+      Run the following command to create a ``quickstart.cpp`` file to contain your
+      application:
+
+      .. code-block:: bash
+
+         touch quickstart.cpp
+
+   .. step:: Create your {+driver-short+} application
+
+      Copy and paste the following code into the ``quickstart.cpp`` file:
+
+      .. literalinclude:: /includes/get-started/quickstart.cpp
+
+   .. step:: Assign the connection string
+
+      Replace the ``<connection string>`` placeholder with the 
+      connection string that you copied from the :ref:`cpp-quick-start-connection-string`
+      step of this guide.
+
+   .. step:: Run your C++ application
+
+      In your shell, run the following commands to compile and run this application:
```

Comment:
I: I am not able to run the app at this point, maybe I created my app in the wrong location?

error: 
```shell
dyld[54430]: Library not loaded: @rpath/libmongocxx._noabi.dylib
  Referenced from: <A3801C50-6B5B-371A-9E65-B8944F9804C9> /Users/rea.rustagi/mongo-cxx-driver-r3.10.1/build/cpp-quickstart/app.out
  Reason: no LC_RPATH's found
[1]    54430 abort      ./app.out
```

Reply:
It seems like this is a somewhat common error on macs (there's a [troubleshooting section](https://www.mongodb.com/docs/languages/cpp/cpp-driver/current/installation/advanced/#fixing-the-library-not-loaded-error-on-macos) on it). I'll do some investigating

Reply:
Added an admonition about this